codex-ui/src/vue/note-list/NoteListItem.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="note">
+    <div class="cover"></div>
+    <div class="footer">
+      <h2 class="noteTitle">{{ props.title }}</h2>
+      <p class="updatedDate">{{ props.updatedAt }}</p>
```

Comment:
undefined class name. P.S. it would be better to use `div` instead of `p` since it is not a paragraph of text